packages/camera/src/components/Layout/index.js

```diff
@@ -119,9 +120,9 @@ function Layout({ backgroundColor, children, left, right }) {
     isPortrait && styles.rightPortrait,
   );
 
-  if (isPortrait) {
+  if (isPortrait && !isPortraitModeVinLayoutView) {
```

Comment:
Tweak to make condition code more easy to readable like, if (isPortraitModeVinLayoutView) {...} else if (isPortrait) {...}

Reply:
Also plz add comments on if and else to describe their task